app/components/mandatarissen/mandataris-card.hbs

```diff
@@ -1,8 +1,13 @@
 <AuCard as |c|>
   <c.header>
     <div class="au-u-flex">
-      <AuHeading @level="2" @skin="4" class="flex-grow">
-        {{this.rol}}
+      <AuHeading
+        @level="2"
+        @skin="4"
+        class="flex-grow"
+        {{did-insert (perform this.setBestuursOrganenForMandataris)}}
+      >
+        {{this.rolWithBestuurOrgaanNames}}
```

Comment:
Role was only used once and depending on the bestuursorgaan names we had to add or remove the dash inbetween so changed it to one getter for both. I simulated it here on how it would look if a user has to wait for 2000ms on the bestuursorgaan names.

@Rahien do we need the loader state? I think it can be shown like in the demo. All data without async is shown on init and the names will come behind. If we do want to show the user that something could be happening behind the name i suggest only a loader for the names but that's maybe a bit silly ;p


https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/36266973/f509931e-9a95-40d7-803b-205ba369b1f4